miden-tx/tests/common/mod.rs

```diff
@@ -78,3 +87,61 @@ impl DataStore for MockDataStore {
         Ok(self.account.code().module().clone())
     }
 }
+
+// HELPER FUNCTIONS
+// ================================================================================================
+pub fn get_new_key_pair_with_advice_map() -> (KeyPair, ([Felt; 4], Vec<Felt>)) {
+    let keypair: KeyPair = KeyPair::new().unwrap();
+
+    let pk: Word = keypair.public_key().into();
+    let pk_sk_bytes = keypair.to_bytes();
+    let to_adv_map = pk_sk_bytes.iter().map(|a| Felt::new(*a as u64)).collect::<Vec<Felt>>();
```

Comment:
We should consider implementing:

```
impl From<KeyPair> for [Word; 2] {
...
}
```

To avoid having to serialize to bytes and then cast to felts.  This should make the interface cleaner. Lets create an issue in crypto to discuss this.  Not required for this PR.